editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackPage.tsx

```diff
@@ -164,35 +165,43 @@ export const FeedbackPage: React.FC<Props> = ({ feedback }) => {
           </Typography>
         </EditorRow>
         <EditorRow>
-          <Feed>
-            <Table stickyHeader sx={{ tableLayout: "fixed" }}>
-              <TableHead>
-                <TableRow
-                  sx={{ "& > *": { borderBottomColor: "black !important" } }}
-                >
-                  <TableCell sx={{ width: 160 }}>
-                    <strong>Type</strong>
-                  </TableCell>
-                  <TableCell sx={{ width: 100 }}>
-                    <strong>Date</strong>
-                  </TableCell>
-                  <TableCell sx={{ width: 340 }}>
-                    <strong>Comment</strong>
-                  </TableCell>
-                  <TableCell sx={{ width: 60 }}></TableCell>
-                </TableRow>
-              </TableHead>
-              <TableBody>
-                {feedback.map((item) => (
-                  <CollapsibleRow
-                    key={item.id}
-                    {...item}
-                    displayFeedbackItems={displayFeedbackItems}
-                  />
-                ))}
-              </TableBody>
-            </Table>
-          </Feed>
+          {feedback.length === 0 ? (
+            <ErrorSummary
+              format="info"
+              heading="No feedback found for this service"
+              message="If you're looking for feedback before January 1, 2024, please contact a PlanX developer."
```

Comment:
nit: in the case of feedback, I think it should read "If you're looking for feedback from more than six months ago, please contact a PlanX developer" - because feedback availability is subject to our data sanitation policy (6 month retention) ! Submission event receipts are always retained (no personal data), and that view is actually filtered from January 2024 forwards for simplicity.

Reply:
Ah yes, I coped this over from the submission log and had made a note to clarify. Thanks for this, will update.